sdk/sdk/src/test/java/org/apache/carbondata/sdk/file/CarbonReaderTest.java

```diff
@@ -1260,8 +1260,8 @@ public int compare(Field o1, Field o2) {
       Assert.assertEquals((double) id / 2, row[4]);
       Assert.assertEquals(true, (boolean) row[5]);
       long day = 24L * 3600 * 1000;
```

Comment:
if these valriables are not used then remove it

Reply:
Done